.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OSDOCS-<jira-issue-id>]: <short-description-of-the-pr> --->
+
+Version(s):
+<!--- Specify the version or versions of OpenShift your PR applies to.  --->
+
+Issue:
+<!--- Add a link to the Bugzilla, Jira, or GitHub issue. --->
+
+Link to docs preview:
+<!--- Add direct links to the preview build.
+The preview will be generated after you open the PR.
+You will need to edit the comment to add the link after the preview builds.--->
+
+Additional information:
+<!--- Optional: Include additional context or expand the description here.
+
+Next steps after opening your PR:
+
+* Ask for peer review from the OpenShift docs team:
+  - Ping the peer review squad (@peer-review-squad) requesting a review in the #forum-docs-review channel (CoreOS Slack workspace)
+    https://coreos.slack.com/archives/C025XCRRH28
+
+* IMPORTANT:
+  - All documentation changes must be verified by a QE team associate before merging.
+  - Squash to one commit before submitting your PR for peer review.
```

Comment:
Should we request folks to review their previews before submitting to make sure it looks as intended? Since they need to wait for it to build to include the link anyway.